src/Videos.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Video from './Video';
+
+const Videos = ({repairVideo}) => {
+    return (
+        <Video 
+          key={repairVideo.id}
+          video={repairVideo.videos}
+          />
```

Comment:
I'd align the the closing tag with Video component above for improved readability.